homeassistant/components/zwave/__init__.py

```diff
@@ -251,7 +247,7 @@ def setup(hass, config):
     options.set_console_output(use_debug)
     options.lock()
 
-    NETWORK = ZWaveNetwork(options, autostart=False)
+    hass.data[ZWAVE_NETWORK] = ZWaveNetwork(options, autostart=False)
```

Comment:
You can keep all code below easier to read (and slightly faster due to skipping unnecessary dict lookup) if you do:

```python
network = hass.data[ZWAVE_NETWORK] = ZWaveNetwork(…)
```